### PR TITLE
Feat/authentication front

### PR DESCRIPTION
**Pull request by MaximeHavez**
# Description
logging with back

# Linked Issues
Authentication

# Changes
Possible to log by the app

# Screenshots
N/A

# Tests
How has it been tested ?

- [x] Manually
- [ ] Automated tests
- [ ] QA
- [ ] Other

# Additional information
Precise any other information

### CODE REVIEW

File: yaki_mobile/lib/presentation/ui/authentication/Authentification.dart [hunk 1]
```diff
@@ -0,0 +1,138 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:yaki/presentation/state/providers/login_provider.dart';
+import 'package:yaki/presentation/ui/shared/views/ButtonApp.dart';
+import 'package:yaki/presentation/ui/shared/views/InputApp.dart';
+import '../shared/views/Header.dart';
```
Comment (PiotrFLEURY) [inline yaki_mobile/lib/presentation/ui/authentication/Authentification.dart:6]: use import packages

File: yaki_mobile/lib/presentation/ui/authentication/Authentification.dart [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+import 'package:yaki/presentation/styles/HeaderTextStyle.dart';
+import 'package:easy_localization/easy_localization.dart';
+
+class Authentification extends ConsumerWidget {
+  bool isChecked = false;
```
Comment (PiotrFLEURY) [inline yaki_mobile/lib/presentation/ui/authentication/Authentification.dart:11]: this boolean should not exist in a consumer widget

File: yaki_mobile/lib/presentation/ui/authentication/Authentification.dart [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+                      padding: const EdgeInsets.only(left: 40, top: 10),
+                      child: Row(
+                        children: [
+                          Checkbox(
+                            value: isChecked,
+                            activeColor: HeaderColor.yellowApp,
+                            onChanged: (bool? value) {
+                              // setState(() {
+                              //   isChecked = value!;
+                              // });
```
Comment (PiotrFLEURY) [inline yaki_mobile/lib/presentation/ui/authentication/Authentification.dart:86]: remove commented code

File: yaki_mobile/lib/presentation/ui/authentication/Authentification.dart [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+
+
+  Authentification({super.key});
+
+  final loginController = TextEditingController();
+  final passwordController = TextEditingController();
+
+  void onPressAuthent(WidgetRef ref, login, password) {
+    final providerlogin = ref.watch(loginProvider);
```
Comment (PiotrFLEURY) [inline yaki_mobile/lib/presentation/ui/authentication/Authentification.dart:21]:
```suggestion
    final providerlogin = ref.read(loginProvider);
```

File: yaki_mobile/lib/presentation/ui/authentication/logged.dart [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class logged extends StatelessWidget {
```
Comment (PiotrFLEURY) [inline yaki_mobile/lib/presentation/ui/authentication/logged.dart:4]: Naming
```suggestion
class Logger extends StatelessWidget {
```

File: yaki_mobile/lib/presentation/ui/shared/views/ButtonApp.dart [hunk 6]
```diff
@@ -1,16 +1,21 @@
 import 'package:flutter/material.dart';
 import '../../../styles/HeaderTextStyle.dart';
```
Comment (PiotrFLEURY) [inline yaki_mobile/lib/presentation/ui/shared/views/ButtonApp.dart:2]: import packages

File: yaki_mobile/lib/presentation/ui/shared/views/InputApp.dart [hunk 7]
```diff
@@ -27,20 +29,20 @@ class InputApp extends StatefulWidget {
...
+  TextEditingController controller = TextEditingController();
+
   @override
   Widget build(BuildContext context) {
-    return TextFormField(
+    return TextField(
+      controller: widget.controller,
+      onChanged: (value) {
+
+      },
```
Comment (PiotrFLEURY) [inline yaki_mobile/lib/presentation/ui/shared/views/InputApp.dart:40]: ??

**Review by PiotrFLEURY — CHANGES_REQUESTED** [review]

**Review by FlorianFlouquet — APPROVED** [review]

**Review by FlorianFlouquet — APPROVED** [review]

**Review by LaurentMag — APPROVED** [review]

File: yaki_mobile/config.json [hunk 8]
```diff
@@ -0,0 +1,4 @@
+{
+  "API_BASE_URL": "http://192.168.43.103:3000/",
+  "CRED_HASH_PASS": "abcdefghijklmnop"
+}
```
Comment (PiotrFLEURY) [inline yaki_mobile/config.json:4]: remove from git

**Review by PiotrFLEURY — CHANGES_REQUESTED** [review]

**Review by PiotrFLEURY — APPROVED** [review]

**Review by HaddoucheRym — APPROVED** [review]

**Pull request merged**